### PR TITLE
EDU-2643: Adds note that encrypted histories are exported with encryption

### PR DESCRIPTION
**Pull request by fairlydurable**
Source of truth: @atihkin

### CODE REVIEW

**Comment by fairlydurable** [comment]
@atihkin Please check and review by end of business 14 June so my team can move forward with our review. Thanks!

File: docs/production-deployment/cloud/export.mdx [hunk 1]
```diff
@@ -38,6 +38,12 @@ The Export feature in Temporal Cloud provides the following benefits:
 
 For pricing information, see [Temporal Cloud Pricing](/cloud/pricing).
 
+:::note
+
+If you have enabled encryption for your Workflow History, your exported data will also be encrypted.
```
Comment (atihkin) [inline docs/production-deployment/cloud/export.mdx:43]: Can we make sure the terminology is consistent across this page and https://docs.temporal.io/production-deployment/data-encryption? So maybe we should say "If you have enabled encryption for the data handled in your Workflow Execution, the exported data will also be encrypted"

Comment (fairlydurable) [inline docs/production-deployment/cloud/export.mdx:43]:
```suggestion
If you have enabled encryption for the data handled in your Workflow Execution, the exported data will also be encrypted.```

Comment (fairlydurable) [reply docs/production-deployment/cloud/export.mdx]: Adopted.

File: docs/production-deployment/cloud/export.mdx [hunk 2]
```diff
@@ -38,6 +38,12 @@ The Export feature in Temporal Cloud provides the following benefits:
 
 For pricing information, see [Temporal Cloud Pricing](/cloud/pricing).
 
+:::note
+
+If you have enabled encryption for the data handled in your Workflow Execution, the exported data will also be encrypted.```
```
Comment (flossypurse) [inline docs/production-deployment/cloud/export.mdx:43]:
```suggestion
If you have enabled encryption for the data handled in your Workflow Execution, the exported data will also be encrypted.
```

**Review by flossypurse — APPROVED** [review]

**Pull request merged**